smarts-imitation/smarts_imitation/utils/adapter.py

```diff
@@ -2,28 +2,39 @@
 import gym
 
 
-def get_observation_adapter(neighbor_mode="LANE"):
-    closest_neighbor_num = 6
+def get_subscribed_features(neighbor_mode, closest_neighbor_num):
+    if neighbor_mode == "LANE":
+        subscribed_features = dict(
+            ego_pos=(2,),
+            heading=(1,),
+            speed=(1,),
+            neighbor_with_lanes=(closest_neighbor_num, 4),  # dist, speed, ttc
+        )
```

Comment:
I have to keep this kind of 4-dimensional ego observation here to make sure some of the previous experiment results can be reproduced.